packages/babel-plugin/README.md

```diff
@@ -0,0 +1,15 @@
+# @reatom/babel-plugin
```

Comment:
need to rename all `babel-plugin` to `macro`

Reply:
or maybe `babel`..

usage
`@reatom/babel/plugin`